packages/@contentlayer/core/src/DataCache.ts

```diff
@@ -36,7 +36,7 @@ export namespace DataCache {
   }: {
     schemaHash: string
     cwd: string
-  }): T.Effect<OT.HasTracer, fs.ReadFileError | fs.JsonParseError | GetContentlayerVersionError, Cache | undefined> =>
+  }): T.Effect<OT.HasTracer, GetContentlayerVersionError, Cache | undefined> =>
```

Comment:
While factually true, I'm not sure whether to merge this change since in this case the error types "accidentally" overlap but are semantically not conveying the same meaning.

Maybe a better a better approach would be to also alias `type LoadCacheFromDiskError = fs.ReadFileError | fs.JsonParseError` to disambiguate them? WDYT?